wire-ios/Wire-iOS/Sources/Authentication/Interface/Helpers/TextFieldValidator.swift

```diff
@@ -104,38 +104,40 @@ extension TextFieldValidator.ValidationError: LocalizedError {
         case .tooShort(kind: let kind):
             switch kind {
             case .name:
-                return "name.guidance.tooshort".localized
+                return L10n.Localizable.Name.Guidance.tooshort
             case .email:
-                return "email.guidance.tooshort".localized
+                return L10n.Localizable.Email.Guidance.tooshort
             case .password, .passcode:
                 return PasswordRuleSet.localizedErrorMessage
             case .unknown:
+                // TODO: - [AGIS] This string doesn't exist, replace it
                 return "unknown.guidance.tooshort".localized
             case .phoneNumber:
-                return "phone.guidance.tooshort".localized
+                return L10n.Localizable.Phone.Guidance.tooshort
             }
         case .tooLong(kind: let kind):
             switch kind {
             case .name:
-                return "name.guidance.toolong".localized
+                return L10n.Localizable.Name.Guidance.toolong
             case .email:
-                return "email.guidance.toolong".localized
+                return L10n.Localizable.Email.Guidance.toolong
             case .password, .passcode:
-                return "password.guidance.toolong".localized
+                return L10n.Localizable.Password.Guidance.toolong
             case .unknown:
+                // TODO: - [AGIS] This string doesn't exist, replace it
```

Comment:
When will this TODO handled?

Reply:
I will handle this as soon as I'm done with the initial migration to the generated struct.